apps/emqx_management/src/emqx_management.appup.src

```diff
@@ -1,6 +1,9 @@
-%% -*-: erlang -*-
+%% -*- mode: erlang -*-
 {VSN,
- [ {"4.3.2",
+ [ {"4.3.3",
```

Comment:
last release 4.3.2 has app version 4.3.2, so there is no need to have 4.3.3 here.
because app 4.3.3 can never be upgraded from 4.3.3